config/v1/types_oauth.go

```diff
@@ -1,37 +1,416 @@
 package v1
 
-import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	operatorsv1alpha1api "github.com/openshift/api/operator/v1alpha1"
+)
+
+///////////////////////////////////////////////
```

Comment:
nit: Generally discourage sections like this (the `//////`), but having a few lines to document the point of the file is good.

Reply:
removed